sycl/test/basic_tests/image.cpp

```diff
@@ -1,12 +1,11 @@
+// UNSUPPORTED: cuda
+// CUDA cannot support SYCL 1.2.1 images.
```

Comment:
Just out of curiosity, what is more correct to say?
(CUDA cannot support SYCL 1.2.1 images due to some issue that can never be resolved) or (CUDA can, but does not support them)?

Reply:
The SYCL/OpenCL and CUDA programming models handle image/sampler objects and surface/texture objects fundamentally differently, making and implementation on CUDA with proper HW support impossible.